addons/godot-next/references/property_info.gd

```diff
@@ -60,3 +60,9 @@ static func new_dictionary(p_name: String, p_hint: int = PROPERTY_HINT_NONE, p_h
 
 static func new_resource(p_name: String, p_hint_string: String = "", p_usage: int = PROPERTY_USAGE_DEFAULT) -> PropertyInfo:
 	return load(SELF_PATH).new(p_name, TYPE_OBJECT, PROPERTY_HINT_RESOURCE_TYPE, p_hint_string, p_usage)
+
+static func new_editor_only(p_name: String):
```

Comment:
Debating whether we should add PROPERTY_USAGE_SCRIPT_VARIABLE to these two new additions. the *_only naming scheme would be a bit of a misnomer, but it implies the *use* of the variable to the user clearly since anything the user can manipulate must necessarily already pertain to scripts. So...they can't NOT make script variables, technically speaking. Might as well make accurate PropertyInfo structs with the same name.

I don't think this impacts any functionality at this moment, but it could in the future if things were ever changed, so we need to be sure we respect that these are actually going to be script variables.

Reply:
for now added `PROPERTY_USAGE_SCRIPT_VARIABLE ` in the long run we can see if it needs to change